skiko/src/commonMain/kotlin/org/jetbrains/skia/paragraph/TextStyle.kt

```diff
@@ -286,6 +285,23 @@ class TextStyle internal constructor(ptr: NativePointer) : Managed(ptr, _Finaliz
         return this
     }
 
+    var baselineShift: Float
+        get() = try {
+            Stats.onNativeCall()
+            TextStyle_nGetBaselineShift(_ptr)
+        } finally {
+            reachabilityBarrier(this)
+        }
+        set(value) {
+            setBaselineShift(value)
+        }
+
+    fun setBaselineShift(baselineShift: Float): TextStyle {
```

Comment:
Guess we do not need setter returning this, as in Kotlin better idiom is using apply and not setter chaining.

Reply:
Agree, but such methods are used for similar fields in this class, so I've implemented it